baseline/OSPS-QA.yaml

```
@@ -62,31 +62,21 @@ criteria:
     security_insights_value: # TODO
 
   - id: OSPS-QA-03
-    maturity_level: 2
+    maturity_level: 1
     criterion: |
-      All released software assets MUST be
-      delivered with a machine-readable list of
-      all direct and transitive internal software
-      dependencies with their associated version
-      identifiers.
+      The code repository MUST contain a dependency 
+      list that accounts for the direct language dependencies
+      when the package management system supports it.
     rationale: |
       Provide transparency and accountability for
-      the project's dependencies, enabling users
-      and contributors to understand the
-      software's dependencies and versions.
+      the project's dependencies by fixing the desired versions
```

```suggestion
      the project's dependencies
```

For level 1, I'm not sure we want to require version pinning (or if we do, we need to be very clear about what is a sufficient strictness). To my mind, listing the dependencies is sufficient for level 1.